templates/repo/shabox_badge.tmpl

```diff
@@ -2,10 +2,12 @@
 	{{if .verification.Verified}}
 		<div title="{{if eq .verification.TrustStatus "trusted"}}{{else if eq .verification.TrustStatus "untrusted"}}{{$.root.i18n.Tr "repo.commits.signed_by_untrusted_user"}}: {{else}}{{$.root.i18n.Tr "repo.commits.signed_by_untrusted_user_unmatched"}}: {{end}}{{.verification.Reason}}">
 		{{if ne .verification.SigningUser.ID 0}}
-			<i class="lock icon"></i>
+			<i class="icons">
+				<i class="lock icon"></i>
+			</i>
 			<img class="ui signature avatar image" src="{{.verification.SigningUser.RelAvatarLink}}" />
 		{{else}}
-			<i title="{{.verification.Reason}}" class="icons">
```

Comment:
Why remove the title?

Reply:
It's the same as parent div, so it isn't necessary to double it here.

Reply:
Which div are you talking about?

Reply:
```
<div title="{{if eq .verification.TrustStatus "trusted"}}{{else if eq .verification.TrustStatus "untrusted"}}{{$.root.i18n.Tr "repo.commits.signed_by_untrusted_user"}}: {{else}}{{$.root.i18n.Tr "repo.commits.signed_by_untrusted_user_unmatched"}}: {{end}}{{.verification.Reason}}">
```

Reply:
https://github.com/go-gitea/gitea/pull/11995/files/7370a09a184929b31bbef36644b8baff76e92eee#diff-f83349a3876d62b50566b1395e993a0aL3